org.codeberg.dimkard.glossaico.json

```diff
@@ -1,8 +1,9 @@
 {
     "id": "org.codeberg.dimkard.glossaico",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.15",
+    "runtime-version": "5.15-21.08",
```

Comment:
you should try 5.15-22.08. probably require llvm14 below.

Reply:
Thanks, tried it here #2